e2e/cluster/local/docker-compose.yaml

```diff
@@ -0,0 +1,41 @@
+version: "3.7"
+
+volumes:
+  kubeconfig:
+    name: e2e-local-kubeconfig
+
+services:
+
+  leader:
```

Comment:
nit: I think server/agent nomenclature is more accurate than leader/worker

Reply:
IMO these are too generic for this purpose.